app/controllers/search/news-items.js

```diff
@@ -96,13 +97,16 @@ export default class SearchNewsItemsControllers extends Controller {
     }
 
     if (row.htmlContent) {
-      copyText += sanitizeHtml(
-        row.htmlContent
-          .replace(/<p>(.*?)<\/p>/gi, '$1\n\n') // Replace p-tags with \n line breaks
-          .replace(/<br\s*[/]?>/gi, '\n') // Replace br-tags with \n line break
-          .trim(), // Trim whitespaces at start & end of the string
-        { allowedTags: [], allowedAttributes: {} } // Remove all remaining tags from the string
-      );
+      copyText +=
+        decodeEntities(
+          sanitizeHtml(
+            row.htmlContent
+              .replace(/<p[^>]*>(.*?)<\/p>/gi, '$1\n\n')  // Replace p-tags with \n line breaks
+              .replace(/<br\s*[/]?>/gi, '\n') // Replace br-tags with \n line break
+              .trim(), // Trim whitespaces at start & end of the string
+            { allowedTags: [], allowedAttributes: {} } // Remove all remaining tags from the string
+          )
+        );
```

Comment:
DIt stukje gebruiken we 3x exact hetzelfde (mits andere input).
Willen we een util maken?

Reply:
Met deze aanpassingen leek me het inderdaad ook wel geen slecht idee. Goed als ik dat volgende sprint zelf even opneem?

Reply:
voor mij goed :+1: